framework/tests/h1h2checker/Issue282.java

```diff
@@ -0,0 +1,15 @@
+import org.checkerframework.framework.testchecker.h1h2checker.quals.*;
+
+public class Issue282 {
+    // :: error: (super.invocation.invalid) :: warning: (inconsistent.constructor.type)
```

Comment:
I would add a short comment that these are just from the j.l.Object constructor - maybe use `@SuppressWarnings` instead of expecting the errors, to make clear that this isn't what this test is about.